src/main/webapp/app/utils/date.utils.ts

```diff
@@ -74,3 +74,7 @@ export function dayOfWeekZeroSundayToZeroMonday(dayOfWeekZeroSunday: number): nu
     }
     return (dayOfWeekZeroSunday + 6) % 7;
 }
+
+export function isDateLessThanAWeekAway(date: dayjs.Dayjs): boolean {
```

Comment:
This function only checks if the date is less than a week *in the future*, doesn't it? Can you also reflect that in it's name?